upstream-community-operators/synopsys/synopsys-operator.2019.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,425 @@
+    mediatype: image/png
+  version: 2019.1.0
+  maturity: stable
+  maintainers:
+  - name: Synopsys, Inc
+    email: software-integrity-support@synopsys.com
+  provider:
+     name: Synopsys, Inc
+  labels:
+     alm-owner-etcd: synopsys-operator
+     operated-by: synopsys-operator
+  selector:
+     matchLabels:
+       alm-owner-etcd: synopsys-operator
+       operated-by: synopsys-operator
+  links:
+   - name: Synopsys Integrity
+     url: https://www.synopsys.com/software-integrity.html
+   - name: Synopsys Operator Documentation
+     url: https://synopsys.atlassian.net/wiki/spaces/BDLM/pages/34373652/Synopsys+Operator
+   - name: Black Duck Datasheet
+     url: https://www.synopsys.com/content/dam/synopsys/sig-assets/datasheets/blackduck-sca-ds-ul.pdf
+   - name: OpsSight Data Sheet
+     url: https://www.synopsys.com/content/dam/synopsys/sig-assets/datasheets/openshift-ds-ul.pdf
+   - name: Black Duck Alert Documentation
+     url: https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/622882/Black+Duck+Alert
+   - name: Synopsys Operator Source Code
+     url: https://github.com/blackducksoftware/synopsys-operator
+  installModes:
+   - type: OwnNamespace
+     supported: true
+   - type: SingleNamespace
+     supported: false
+   - type: MultiNamespace
+     supported: false
+   - type: AllNamespaces
+     supported: true
+  install:
+    strategy: ReplicationController
```

Comment:
OLM currently only supports `deployments` for operators. Is there a reason why a ReplicationController is preferred?

Reply:
@SamiSousa since we are supporting customers using openshift 3.6 and 3.7, I don't think deployments are available in those version.

Reply:
updated locally. We used replicationcontrollers for 3.6 compatibility, but we can doc that exception since most users will be on 4.0 using these Operator files